cypress/integration/reserchGroupCanCreateArticle.spec.js

```diff
@@ -0,0 +1,50 @@
+describe("Research Group can post article", () => {
+  beforeEach(function() {
+    cy.server();
+  });
+
+  it("successfully", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:successful_saving_article_response.json"
+    });
+    cy.research_group_login("climate_harvard@mail.com", "password");
+    cy.contains("Hello climate_harvard@mail.com!");
+    cy.get("#login-button").should("not.exist");
+    cy.get("#login-form").should("not.exist");
+    cy.get("#create-article-button").click();
+    cy.get("#create-article-form").within(() => {
```

Comment:
These tests should be extracted. Testing for the visibility of the button and form is outside of scope for this test.

Reply:
We extracted the out of scope tests